framework/draw/internal/fontsengine.cpp

```
@@ -510,6 +517,50 @@ IFontFace* FontsEngine::createFontFace(const io::path_t& path) const
     return new FontFaceDU(origin);
 }
 
+IFontFace* FontsEngine::fontFace(const FontDataKey& dataKey, Font::Type type, int pixelSize, bool isSymbolMode) const
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)